docs/cloud-providers.md

```diff
@@ -1,43 +1,18 @@
 ---
-title: Connect Your Cloud Providers
+title: Configure Your Cloud Provider Account
 toc: true
 weight: 230
 indent: true
 ---
 
-# Connect Your Cloud Provider
+# Configure Your Cloud Provider Account
 
-In order for Crossplane to be able to manage resources across all your clouds, you will need to add your cloud provider account credentials to Crossplane.
-Use the links below for specific instructions to add each of the following cloud providers:
+In order for Crossplane to be able to manage resources across all your clouds,
```

Comment:
should we rephrase `across all your clouds` to `in a specific cloud provider`

Reply:
How about:
> "In order for Crossplane to be able to manage resources on a cloud provider,
you will need to add your account credentials for that provider to Crossplane.
Use the links below for specific instructions to configure accounts that
can be used throughout the guides:"

Reply:
I don't want to mention _adding to Crossplane_ since the links do not include that at all. They just _create_ the accounts, a process that happens completely on provider UI. Then later in the guides we _add_ those credentials.